### PR TITLE
data: Manazsharma

### PR DESCRIPTION
**Pull request by Manazsharma**
Making my first open-source contribution

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5904"></a>

### CODE REVIEW

File: data/Manazsharma.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"bio": "Frontend Devloper| Open Source Enthusiast| Intrested in Web3 & Devops ",
+	"links": [{
+			"name": "Follow me on Twitter",
+			"url": "https://twitter.com/manassharma__",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Follow me on LindkedIn",
+			"url": "https://www.linkedin.com/in/manas-sharma-a49295220/",
+			"icon": "FaLinkedIn"
```
Comment (kumarsonsoff3) [inline data/Manazsharma.json:13]: You probably be using an invalid icon here. It should be `FaLinkedin`.

```suggestion
			"icon": "FaLinkedin"
```

File: data/Manazsharma.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		},
+		{
+			"name": "GitHub",
+			"url": "https://github.com/Manazsharma",
+			"icon": "FaGitHub"
```
Comment (kumarsonsoff3) [inline data/Manazsharma.json:18]: Also, this should be `FaGithub`.

```suggestion
			"icon": "FaGithub"
```

**Review by kumarsonsoff3 — COMMENTED** [review]
Hey Manaz, Congrats on the first open-source contribution. :tada: 

Minor fixes required. 🛠

**Review by SurajPratap10 — APPROVED** [review]
Congrats on your successful contribution.  LGTM!!

**Comment by Manazsharma** [comment]
Issues Fixed.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**